### ISSUE TITLE
Lyrics not loading in

### ISSUE THREAD

**Issue by JonaVDM** [ShyykoSerhiy/vscode-spotify-lyrics#8]
Had an issue viewing lyrics in the Spotify vs-code extension, It would just not do anything.

When I tired visiting the website in the browser with for example https://vscode-spotify-lyrics.azurewebsites.net/lyrics?artist=Foo%20Fighters&title=Best%20of%20you I got this message 
![image](https://user-images.githubusercontent.com/32264711/87716867-c0702280-c7af-11ea-891d-c34b0f225a0f.png)

When creating a new app and running it locally with the same url params it does seem to work.
Maybe a problem with your access token?

**Comment by ShyykoSerhiy** [comment]
Yeah, it looks like Genius started blocking requests made from my server's IP. I'll need to rewrite the logic to make requests to Genius from the extension itself to fix it. Thanks @JonaVDM for opening the issue

**Comment by rileyhutton** [comment]
Presume this isn't going to be fixed? If not I maaayyy try and work my own solution
